src/views/app.view.tsx

```diff
@@ -75,9 +77,10 @@ function App({
   useAppStyles();
 
   React.useEffect(() => {
+    onLoadEnv();
     onCheckHermezStatus();
     onLoadFiatExchangeRates();
```

Comment:
Is this working now? As it's only called once, it could be possible that the `env` is still `undefined`. I think that it's more secure to create another hook and be sure that we have the `env` properly loaded:

```ts
React.useEffect(() => {
  if (env) {
    onCheckHermezStatus(env);
    onLoadFiatExchangeRates(env);
  }
}
```

Reply:
Yes, it is working, but you have a point there. Changed.